trackma/lib/libkitsu.py

```diff
@@ -560,8 +560,8 @@ def _parse_info(self, media):
             # For now I'm just picking the romaji title in these cases.
             'title':       attr['titles'].get('en_jp') or attr.get('canonicalTitle') or attr['titles'].get('en'),
             'total':       total or 0,
-            'image':       attr['posterImage'] and attr['posterImage']['small'],
-            'image_thumb': attr['posterImage'] and attr['posterImage']['tiny'],
+            'image':       attr['posterImage'] and attr['posterImage'].get('small'),
```

Comment:
Should we have a fallback to 'original' if 'small' cannot be found?

Reply:
`image` is used by the details dialog. How small is the small thumb? And yes I think a fallback to original would be sensible as the details dialog can be as big as you want to.